war/src/main/webapp/scripts/hudson-behavior.js

```diff
@@ -753,7 +753,8 @@ function labelAttachPreviousOnClick() {
 
 function helpButtonOnClick() {
     var tr = findFollowingTR(this, "help-area", "help-sibling") ||
-             findFollowingTR(this, "help-area", "setting-help");
+             findFollowingTR(this, "help-area", "setting-help") ||
+            findFollowingTR(this, "help-area");
```

Comment:
Should this be indented one more space?
```suggestion
             findFollowingTR(this, "help-area");
```